Makefile

```diff
@@ -2,7 +2,7 @@
 DEIS_REGISTRY ?= $(DEV_REGISTRY)
 IMAGE_PREFIX ?= deis
 COMPONENT ?= workflow
-VERSION ?= git-$(shell git rev-parse --short HEAD)
+VERSION ?= $(shell git rev-parse --short HEAD)
```

Comment:
okay let's do that for the client, but let's keep in the `git-` convention for workflow, because that's the current version convention for [the](https://github.com/deis/builder/blob/master/Makefile#L15) [other](https://github.com/deis/postgres/blob/master/Makefile#L9) [components](https://github.com/deis/router/blob/master/Makefile#L4-L5).